examples/elastic_wave.cpp

```diff
@@ -85,18 +111,22 @@ int main( int argc, char* argv[] )
                 u( pid, d ) = a * std::exp( -arg ) * comp;
                 v( pid, d ) = 0.0;
             }
+            // Density
             rho( pid ) = rho0;
         };
         particles->updateParticles( exec_space{}, init_functor );
 
-        auto bc = CabanaPD::createBoundaryCondition<memory_space>(
-            CabanaPD::ZeroBCTag{} );
-
+        // ====================================================
+        //                   Simulation run
+        // ====================================================
         auto cabana_pd = CabanaPD::createSolverElastic<memory_space>(
             inputs, particles, force_model, bc );
         cabana_pd->init_force();
         cabana_pd->run();
 
+        // ====================================================
+        //                      Outputs
```

Comment:
Linear profile output

Reply:
I would prefer to keep section titles general. I could add a comment below, as follows:
// Output displacement along the x-axis

What do you think?

Reply:
Looks fine